src/snippets/switch.md

```diff
@@ -0,0 +1,57 @@
+---
+description: Switch Statement Snippet
+---
+
+# Switch
+
+Using `switch` statements in AssemblyScript
```

Comment:
Could mention that switch conditions currently implicitly convert to `u32`, i.e. switching over  `string`s or similar is not yet supported. If using `i32`s, a more technical detail would be that using values greater than zero has better codegen, because `br_table`s are unsigned, so a `-1` for example would not tablify well with otherwise positive values (it does work, though, but code is not optimal).